src/CMakeLists.txt

```diff
@@ -55,9 +51,18 @@ set(gui_sources
   PARENT_SCOPE
 )
 
-set(cli_sources
-  gz.cc
-  cmd/ModelCommandAPI.cc
+set(material_sources
+  rendering/MaterialParser/MaterialParser.cc
+  rendering/MaterialParser/ConfigLoader.cc
+)
+
+set(network_sources
```

Comment:
Is there a reason for moving this from line 38? I'd rather minimize the diff if possible.

Reply:
Oh just thought of sorting them alphabetically, reverted back for minimal diff.